### PR TITLE
added "how we vote"

### PR DESCRIPTION
**Pull request by DanM3rcurius**
one more item done on EPIC task #61 , needs to be prettied up with some flow diagram but for now it works!

### CODE REVIEW

File: Contribute.md [hunk 1]
```diff
@@ -24,6 +24,19 @@ This section includes ways to get started with your open source project. Include
 * [Blog](https://medium.com/@secretdecks)         
 * Anything else: see [Website](www.secretdecks.com)
 
+# How we vote
+
+- Any core member can create a poll
+- Before a poll can be created in Discord, it has to (manually) receive two peer approvals; hence a github issue [has to be raised](https://github.com/SecretDecks/Documentation/issues/new/choose), giving a short summary of the poll and asking for peer review. A template for the voting report can be found [HERE](https://github.com/SecretDecks/Documentation/issues/new/choose)
```
Comment (stephen-rowan) [inline Contribute.md:30]: Rewrite -
"Before a poll can be created in Discord it has to manually receive two peer approvals. A github issue [has to be raised](https://github.com/SecretDecks/Documentation/issues/new/choose) which gives a short summary of the poll and requests a peer review. A template for the voting report can be found [HERE](https://github.com/SecretDecks/Documentation/issues/new/choose)."

File: Contribute.md [hunk 2]
```diff
@@ -24,6 +24,19 @@ This section includes ways to get started with your open source project. Include
...
+- in case the vote result has a large impact, a proposal needs to written upfront and the voting timeframe has to be 7 days minimum
+- in case vote result has a small impact, poll can be created with a minimum timeframe of 48h
+- poll answers/results need to be documented in the issue raised before, as well as approved by two peers
+- any change management needs to reference the issue in GitHub when creating additional items like discussions or other issues
```
Comment (stephen-rowan) [inline Contribute.md:34]: Rewrite -
"- If  the vote result has a large impact a proposal will need to be written beforehand and the voting timeframe has to be 7 days minimum.
- If the vote result has a small impact a poll can be created with a minimum timeframe of 48 hours.
- Poll answers & results need to be documented in the issue raised beforehand and be approved by two peers.
- Any change management needs to reference the issue in GitHub when creating additional items like discussions or other issues.

**Review by tamagit80 — APPROVED** [review]

File: Contribute.md (hunk 1 above)
Comment (DanM3rcurius) [reply Contribute.md:30]: changed! 🙏🏿

File: Contribute.md (hunk 2 above)
Comment (DanM3rcurius) [reply Contribute.md:34]: small edits done, thank you Sir 👍🏿

**Pull request merged**